File: frontend/src/app/shared/models/notification.ts
import { Organization } from './organization';
import { User } from './user';

export class Notification {
  constructor(
    public id: number,
    public text: string,
    public createdAt: Date,
    public organizationId: number,
    public organization: Organization,
    public userId: number,
    public user: User) {
  }
}

File: backend/Watcher.Common/Dtos/Notification.cs
using System;
using Watcher.Common.MoveToFrontend;

namespace Watcher.Common.Dtos
{
    [ExportClassToTypescript]
    public class Notification : Entity<int>
    {
        public override int Id { get; set; }
        public string Text { get; set; }
        public DateTime CreatedAt { get; set; }

        public int? OrganizationId { get; set; }
        public Organization Organization { get; set; }

        public int UserId { get; set; }
        public User User { get; set; }
    }
}
